CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 * Enhancements
     * Added ability to include command name placeholders in the message printed when trying to run a disabled command.
         * See docstring for ``disable_command()`` or ``disable_category()`` for more details.
+* Potentially breaking changes
+    * The following commands now write to stderr instead of stdout when printing an error. This will making catching
+    errors easier in pyscript.
+        * ``do_help()`` - when no help information can be found
```

Comment:
Recommend calling out the format change of this output as well as a potentially breaking change in case someone has unit or regression tests which depend on the existing format